selftests/functional/test_statuses.py

```diff
@@ -0,0 +1,291 @@
+import json
+import os
+import shutil
+import tempfile
+import unittest
+
+from avocado.utils import process
+from avocado.utils import script
+
+
+basedir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..')
+basedir = os.path.abspath(basedir)
+
+AVOCADO = os.environ.get("UNITTEST_AVOCADO_CMD", "./scripts/avocado")
+
+TEST_CONTENT = """
+import os
+import sys
+from avocado import Test
+
+class StatusTest(Test):
```

Comment:
Would you please re-consider putting this into `selftests/.data`? It is really useful test for development and that way one would be able to execute it while working on some feature.